staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -912,6 +912,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
+	// podPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
```

Comment:
Does not contain the description I was expecting to see about how this feature works.  I would expect 1-2 sentences describing how a value passed to this flag impacts the actual shutdown flow.

Reply:
Does something like this work?
```
        // +optional
        ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
        // podPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
+       // Each entry in the array represents the graceful shutdown time a pod with a priority class value that 
+       // lies in the range of that value and the next higher entry in the list when the node is shutting down.
+       // For example:
+       // |Pod priority class value|Shutdown period|
+       // |------------------------|---------------|
+       // | 100000                 |10 seconds     |
+       // | 10000                  |180 seconds    |
+       // | 1000                   |120 seconds    |
+       // | 0                      |60 seconds     |
+       // With the above configuration set, any pod with priority value >= 100000 will get
+       // 10 seconds to stop, any pod with value >= 10000 and < 100000 will get 180
+       // seconds to stop, any pod with value >= 1000 and < 10000 will get 120 seconds to stop.
+       // All other pods will get 60 seconds to stop.
        // +featureGate=GracefulNodeShutdown
        // +featureGate=PodPriorityBasedGracefulShutdown
        // +optional
```

Reply:
Sorry for missing it, it's done now.

Reply:
I added a bunch of comments to the other type (internal api) but they apply here.